backend/src/common/services/azure/azure-ai.service.ts

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+
+import { type BaseConfig } from '~/common/config/base-config.package.js';
+import { HttpError } from '~/common/http/http.js';
+
+import { type AzureGetAvatarResponseDto } from './types/types.js';
+
+class AzureAIService {
```

Comment:
Looks like we need

- voices
- avatar info
- text to speech
- get full rendered video

Reply:
as for now it is not merged yet avatar info, I return voices filtered by language EN, the URL from result of text to speech, and I added two mwthods to create a batchSynthesisVideo, and get the url of that video, we should receive SSML script like here: https://learn.microsoft.com/en-us/azure/ai-services/speech-service/text-to-speech-avatar/avatar-gestures-with-ssml#supported-prebuilt-avatar-characters-styles-and-gestures, where we can add gestures